reporter/html.js

```diff
@@ -747,13 +747,19 @@ QUnit.testDone(function( details ) {
 	}
 });
 
-if ( !defined.document || document.readyState === "complete" ) {
+if ( defined.document ) {
+	if ( document.readyState === "complete" ) {
+		if ( window.define && window.define.amd ) {
+			QUnit.config.autostart = false;
```

Comment:
We could certainly make it do that. The thought was that if the loading isn't complete, then you probably didn't load QUnit through AMD, even if AMD is available. But I think it's probably ok to assume that if AMD is available that you'll always want auto-start disabled.